### ISSUE TITLE
My mods doesn't appear in smapi installer's config

### ISSUE THREAD

**Issue by Leuynn** [ZaneYork/SMAPI-Android-Installer#620]
![Screenshot_2023-11-17-19-14-40-238_com zane smapiinstaller](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/136723448/676d6702-5a4b-4598-89b9-2a8202f06122)
mods works fine, i put the all of my mods in the data folder, and it seems that the smapi installer can't read it? is there some way to fix this or can i change the mods path?

**Comment by Warilized** [comment]
It's a known issue because the mod path has been changed due to Android 13 Scooped storage restriction.
The installer still use the old mod path `StardewValley/Mods`, while the SMAPI uses `android/data/com.zane.stardewvalley/files/mods`/

**Comment by Leuynn** [comment]
> It's a known issue because the mod path has been changed due to Android 13 Scooped storage restriction. The installer still use the old mod path `StardewValley/Mods`, while the SMAPI uses `android/data/com.zane.stardewvalley/files/mods`

so it's impossible to change it for now?

**Comment by Warilized** [comment]
> so it's impossible to change it for now?

Yeah, unless zane updated the mod. It's Impossible to edit config files thru the installer.
You can however edit it manually using either a PC or an apps that can access data folders (like MiXplorer or FV File Explorer) and edit it from `android/data/com.zane.stardewvalley/files/mods`

There's a "how to access android/data" on the net, though it might not works because mine only works for a short time.

**Comment by Zencee** [comment]
if you really wanna use the installer to edit config using your android phone only then there's a way but it's kinda tedious,

1. make sure you have the correct mod path on your smapi installer which is `StardewValley/Mods`
2. make sure you have the config.json files inside your Mods you wanna edit, this means you've already run the game at least once with the mod installed.
3. Next is to find the mods you want to edit inside `Com.zane.stardewvalley/files/Mods` and copy or move them to `StardewValley/Mods` so that you could edit them thru smapi installer.
4. after editing them to your liking, move the mods back to `Com.zane.stardewvalley` and run your game.

as I said this process is kinda tedious but its worth it especially if you want to edit your virtual keyboard using the graphics function on smapi installer, otherwise you can use some third party apps to edit your Mods like warilized said.
